core/ops.rs

```diff
@@ -22,91 +20,78 @@ use std::pin::Pin;
 use std::ptr::NonNull;
 use std::rc::Rc;
 use std::rc::Weak;
-use std::task::Context;
-use std::task::Poll;
 use v8::fast_api::CFunctionInfo;
 use v8::fast_api::CTypeInfo;
 
-/// Wrapper around a Future, which causes that Future to be polled immediately.
-///
-/// Background: ops are stored in a `FuturesUnordered` structure which polls
-/// them, but without the `OpCall` wrapper this doesn't happen until the next
-/// turn of the event loop, which is too late for certain ops.
-pub struct OpCall<T>(MaybeDone<Pin<Box<dyn Future<Output = T>>>>);
+pub type RealmIdx = usize;
+pub type PromiseId = i32;
+pub type OpId = usize;
```

Comment:
```suggestion
pub type RealmIdx = u32;
pub type PromiseId = i32;
pub type OpId = u16;
```

Let's make the size smaller. `u16` is enough for OpId and `u32` for RealmIdx. Also change op_id in OpCtx.

Reply:
Changed both to `u16`